src/Command/ImportScorerCommand.php

```diff
@@ -0,0 +1,184 @@
+                                Stopwatch $stopwatch, AdapterInterface $provider)
+    {
+        $this->encoder = $encoder;
+        $this->objectManager = $objectManager;
+        $this->stopwatch = $stopwatch;
+        $this->dataProvider = $provider;
+        parent::__construct('app:import-scorer');
+    }
+
+    /**
+     *  configure options and argument of the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Import scorers data from a CSV file')
+            ->setHelp('CSV file HEADERS: \'username,password\'');
+        $this->addOption(
+            'force',
+            'f',
+            InputOption::VALUE_NONE,
+            'Causes data ingestion to be applied into storage'
+        );
+        $this->addOption(
+          'overwrite',
+            'o',
+            InputOption::VALUE_NONE,
+            'overwrite Scorer in database'
+        );
+        $this->addArgument(
+            'path',
+            InputArgument::REQUIRED,
+            'Source path to ingest from'
+        );
+    }
+
+    /**
+     * Initialize variables
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @throws \Exception
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->input = $input;
+        $this->output = $output;
+        $this->path = $input->getArgument(self::ARGUMENT_PATH);
+        $this->symfonyStyle = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * import scorer from data file to the database
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->stopwatch->start('import');
+        $this->symfonyStyle->title('Human-Scoring Import');
+        if (!$input->hasParameterOption(self::OPTION_OVERWRITE)) {
+            $output->writeln(['if you want overwrite scorer data in database use flag \'--overwrite\'', '']);
+        }
+        try {
+            $data = $this->dataProvider->getIterator($this->path);
+            $this->persist($data);
+            $this->flush();
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+            return 1;
+        }
+            $event = $this->stopwatch->stop('import');
+            $this->showStopwatchData($event->getMemory(),$event->getDuration ());
+            return 0;
+    }
+
+
+    /**
+     * if there is option '--overwrite' persist
+     * persist scorers
+     *
+     * @param $data
+     */
+    private function persist($data)
+    {
+        foreach ($data as $scorer) {
+            $dbScorer = $this->objectManager->getRepository(Scorer::class)->findOneBy(['username' => $scorer['username']]);
+            if (!$this->input->hasParameterOption(self::OPTION_OVERWRITE)) {
+                if (!($dbScorer instanceOf Scorer)) {
```

Comment:
And please try to avoid nesting if cases, its hard to follow. You can check some other clean code instructions also: https://github.com/jupeter/clean-code-php#avoid-nesting-too-deeply-and-return-early-part-1